src/routes/docs/contributing/code-style.svx

```diff
@@ -10,32 +10,97 @@
+- Do not use `#region` directives
+- Align `while();` on the same line of curly bracket of `do..while`
+- Use the prefixes `Is`, `Has`, `Have`, `Any`, `Can` or similar keywords for boolean names and method names if applicable.
+- Do not use `Enum` or `Flag(s)` suffixes/prefixes for Enum names.
+- Align curly brackets on new lines
+- Fields, properties, methods and constructors must be separated by one line
+  except for property backing fields that are not separated by any amount of lines
+- Do not manually invoke event handling functions or commands
+- Do not use `public` modifiers in interface's properties and functions
+- Do not prefix interface name before implementing function/property unless necessary
+- Define constant types in `Constants` class following its structure
+- Mark classes as `abstract` or `sealed` if applicable.
+- Use single line return `=>` for simple functions
+- Complex classes and interfaces must be documented
+- Always create new `EventArgs` classes for event arguments
+- Add letter suffix to number values: `0.1d`, `100000000000L`, `1.1f`
+- You are obliged to use [XAML Styler](https://marketplace.visualstudio.com/items?itemName=TeamXavalon.XAMLStyler2022) for styling XAML
 
-3. **Don't use** Hungarian Notation.
+3. Class structure
+The following presents how a class structure should be ordered:
 
-4. **Don't use** underscores (`_`), hyphens (`-`), or any other non-alphanumeric characters.
+- private/protected/public (internal) static fields
+- readonly private/protected/public (internal) static fields
+- private/protected/public (internal) static properties
 
-5. **Don't use** CAPITAL LETTERS in any names.
+- readonly private/protected/public (internal) fields
+- private/protected/public (internal) fields
 
-6. **Use** predefined type names like `int`, `string`, etc. instead of `Int32`, `String`.
+- private/protected/public (internal) service property
 
-7. **Do** prefix the names of interfaces with `I`.
+- private/protected/public (internal) property backing fields
+- private/protected/public (internal) properties
+- private/protected/public (internal) UI properties
+- private/protected/public (internal) ICommand properties;
 
-8. **Do** align curly brackets vertically.
+- private/protected/public (internal) constructor
+
+- Command implementation functions (private only)
+
+- public (virtual/override) functions/methods (with parameters amount descending)
+- private functions/methods
+- protected (virtual/override) functions/methods (with parameters amount descending)
+
+- public/private static functions/methods
+
+- public/protected abstract functions/methods
+
+- public operators
+
+- Assertions e.g. AssertNotDisposed(), AssertInitialized(), AssertStreamOpened()
+- IDisposable.Dispose() function
+
+4. Interface structure
+The following presents how an interface structure should be ordered:
+
+- properties
+- functions (with parameters amount descending)
+- functions (without parameters)
```

Comment:
Why not sort functions by name and than sort by parameter count (no parameters being first)?